cmd/crossplane/main.go

```diff
@@ -59,7 +65,11 @@ func main() {
 		extCmd = app.Command("stack", "Perform operations on stacks")
 
 		// stack manage - adds the stack manager controllers and starts their reconcile loops
-		extManageCmd = extCmd.Command("manage", "Manage stacks (run stack manager controllers)")
+		extManageCmd                     = extCmd.Command("manage", "Manage stacks (run stack manager controllers)")
+		extManageTenantKubeconfig        = extManageCmd.Flag("tenant-kubeconfig", "The absolute path of the kubeconfig file to Tenant Kubernetes instance. Setting this will activate host aware mode of Stack Manager").String()
+		extManageHostControllerNamespace = extManageCmd.Flag("host-controller-namespace", "The namespace on Host Cluster where install and controller jobs/deployments will be created (required for host aware mode, ignored otherwise)").String()
```

Comment:
Should we set a default namespace? I believe adding `.Default('crossplane-system')` would do that.

Reply:
Host cluster is typically expected to be shared across tenants, which means host controller namespace should have some unique identifier (e.g. `tenant-123-controllers`). So, I don't think it make sense to have a default here.